web-jehlomat/src/Components/Inputs/DateTimeInput/TimeInput.tsx

```diff
@@ -0,0 +1,89 @@
+import { FC, InputHTMLAttributes, useState } from 'react';
```

Comment:
can we reuse DateInput here?

Reply:
There is a different type of picker used, so it is looking similar to figma's design.
I can merge it into one component, but that way there would probably be some condition differentiating time vs date.